e3db/tests/test_integration.py

```diff
@@ -76,7 +81,10 @@ def setup_class(self):
         client3_public_key, client3_private_key = e3db.Client.generate_keypair()
         client3_public_signing_key, client3_private_signing_key = e3db.Client.generate_signing_keypair()
         client3_name = "client_{0}".format(binascii.hexlify(os.urandom(16)))
-        test_client3 = e3db.Client.register(token, client3_name, client3_public_key, api_url=api_url)
+        test_client3 = e3db.Client.register(token, client3_name, client3_public_key, api_url=api_url,
```

Comment:
not blocking, may be nice to spell out backup=False

Reply:
Done